packages/types/src/http/httpHandlerInitialization.ts

```diff
@@ -90,4 +90,9 @@ export interface FetchHttpHandlerOptions {
    * these limitations before enabling keepalive.
    */
   keepAlive?: boolean;
+
+  /** 
+   * A string indicating whether credentials will be sent with the request always, never, or only when sent to a same-origin URL. Sets request's credentials.
+   */
+  credentials?: "include" | "omit" | "same-origin" | undefined | string;
```

Comment:
Nit: maybe credentialsMode instead?

Reply:
`credentials` matches the property on the Fetch request itself. I think it makes sense to keep this property name consistent with how it's used on the request.